.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "nuget/helpers/lib/NuGet.Client"]
+	path = nuget/helpers/lib/NuGet.Client
+	url = https://github.com/NuGet/NuGet.Client
```

Comment:
Can you elaborate on the need on using `NuGet.Client` through a submodule? Is it not distributed in a more standard way?

Reply:
I'll add a detailed comment to the PR description shortly, but the short version is that the functionality required isn't available as a NuGet package or otherwise that can be consumed by a Linux container (it's currently only available as a net472 Windows console app and the NuGet team doesn't want to support any other configuration), so to avoid having a complete copy of the NuGet code here or elsewhere, it was much simpler to use a submodule and only re-create the project files with the relevant changes.